### DISCUSSION TITLE
Kibana and logstasg shut down

### DISCUSSION THREAD

**Discussion by d1vers3** [telekom-security/tpotce#1438] [Q&A]
When you start Tpot, there is no Kibana service on the main screen. After reloading the page multiple times, dps.sh reports that Kibana is running. However, after the next reboot the Tpot drops. What to do?
![изображение_2023-11-18_173153271](https://github.com/telekom-security/tpotce/assets/101476639/23723b4a-f105-4949-aba1-d3e9af965c3f)
![изображение_2023-11-18_173210991](https://github.com/telekom-security/tpotce/assets/101476639/33295b37-49bc-478a-b477-8bea7da1a101)
![изображение_2023-11-18_173226153](https://github.com/telekom-security/tpotce/assets/101476639/900f6265-0e2f-4f91-aabb-4fabee589cbb)

**Comment by inflamedpitbull** [comment]
Make sure you are running Debian 11. I had the exact same issue and after changing my image to Debian 11, everything worked fine.
